packages/cli/src/templates/expo.ts

```diff
@@ -0,0 +1,277 @@
+import chalk from "chalk";
+import { Listr } from "listr2";
+import { existsSync } from "node:fs";
+import { writeFile } from "node:fs/promises";
+import { TemplateOptions } from "../commands/create";
+import { Template } from "./template";
+
+import { exec } from "node:child_process";
+import { promisify } from "node:util";
+const execAsync = promisify(exec);
+export interface ExpoContext {
+  options: TemplateOptions;
+}
+
+export const Expo: Template = {
+  name: "expo",
+  description: "A new expo typescript project with gluestack-ui renderer",
+  async handler(options) {
+    await new Listr<ExpoContext>([
+      {
+        title: "Create directory",
+        task: async ({ options: { cwd } }, task) => {
+          task.title += ` ${cwd}`;
+          if (existsSync(cwd)) {
+            throw new Error(`Directory ${cwd} already exists`);
+          }
+          await execAsync(
+            `npx create-expo-app ${cwd} -t expo-template-blank-typescript`,
```

Comment:
This does not work if the user has never installed the `create-expo-app` package.

I advise doing this:


```suggestion
            `npx -y create-expo-app ${cwd} -t expo-template-blank-typescript`,
```